src/main/java/ca/corefacility/bioinformatics/irida/model/announcements/Announcement.java

```diff
@@ -48,10 +48,17 @@
     @Column(name = "created_date", updatable = false)
     private Date createdDate;
 
+    @Column(name = "title")
+    private String title;
```

Comment:
Since this field is not null in the liquibase please add `@NotNull` here as well please.

Reply:
Sorry I missed this. Please see bd781b5750cdd53260d15238c2e6a9e398c09c1d.